### ISSUE TITLE
ovnkube.log and ovnkube-master.log shouldn’t be in /var/log/openvswitch

### ISSUE THREAD

**Issue by girishmg** [openvswitch/ovn-kubernetes#610]
The log files for ovnkube-node and ovnkube-master daemons are logged under
/var/log/openvswitch. However, when logrotate rotates the log files under /var/log/openvswitch folder daily, the ovnkube daemon cannot log anymore to the new log files. 

```
    0 Feb 11 06:25 ovnkube.log
   20 Feb 10 06:25 ovnkube.log.1.gz
   20 Feb  9 06:25 ovnkube.log.2.gz
   20 Feb  8 06:25 ovnkube.log.3.gz
  13K Feb  7 22:06 ovnkube.log.4.gz
```
After Feb 7th, there is nothing being captured. 

```
# lsof -Pnp `pidof ovnkube` |grep log
ovnkube 60931 root    4w      REG                8,2   141325 14162215 /var/log/openvswitch/ovnkube.log.1 (deleted)
```

We should just log those files under /var/log?

**Comment by shettyg** [comment]
Good catch. One option is for ovnkube to catch the signal and re-open the log file.

**Comment by girishmg** [comment]
@shettyg right we don't have a control AF_UNIX socket. I agree that it is better to gather all the log files under  /var/log/ovn-kubernetes

**Comment by girishmg** [comment]
Fixed in c1e1ca6aac2554117e4bd4613ffd6d5b1940e62e, so closing
